cli/js/lib.deno.ns.d.ts

```diff
@@ -1572,10 +1572,16 @@ declare namespace Deno {
     /** Defaults to `false`. If set to `true`, will append to a file instead of
      * overwriting previous contents. */
     append?: boolean;
+    /** Defaults to `false`. If set to `true`, no file, directory, or symlink is
+     * allowed to exist at the target location. When createNew is set to `true`,
+     * create is ignored. */
```

Comment:
Copy the doc string from Rust - you left out the bit about it being atomic. 

What error is thrown if a file already exists?

Reply:
In the PR description: "we can ensure that errors triggered by violation of a createNew option are always AlreadyExists on all platforms. We now do this...in op_open."

Am happy to include more of this info in the doc comments. But I'll wait for more feedback about this (and its sister PRs, pushing shortly) and will return to the doc comments at the end.

Reply:
Updated the doc-comment for createNew here (and on the other PRs too).